packages/patterns-react/src/patterns/LeadSpaceCentered/README.md

```diff
@@ -0,0 +1,84 @@
+# Lead Space Centered
+
+> The Lead Space Centered pattern is to be utilized within IBM.com.
+
+## Getting started
+
+Here's a quick example to get you started.
+
+```scss
+// yourapplication.scss
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+```
+
+> 💡 Only import font's once per usage
+
+```javascript
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { LeadSpaceCentered } from '@carbon/ibmdotcom-patterns-react';
+import 'yourapplication.scss';
+import '@carbon/ibmdotcom-styles/scss/patterns/leadspace-centered/_leadspace-centered.scss';
+function App() {
+  return <LeadSpaceCentered />;
+}
+ReactDOM.render(<App />, document.querySelector('#app'));
+```
+
+> 💡 Don't forget to import the leadspace styles from
+> [@carbon/ibmdotcom-styles](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/styles).
+
+#### Feature Flags
+
+To utilize the following features, set the following variable's to `true` within
+your `.env` file or your application build settings.
+
+```
+DDS_LEADSPACE_CENTERED=true
+```
+
+> See
+> [feature-flags.md](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/patterns-react/docs/feature-flags.md)
+> and
+> [.env.example](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/patterns-react/.env.example)
+> for more information
+
+## Variations
+
+| Name              | Description                              |
+| ----------------- | ---------------------------------------- |
+| `white`/`default` | White theme applied to pattern           |
+| `g100`            | Gray 100 (g100) theme applied to pattern |
```

Comment:
@annawen1 is this a prop now?

Reply:
Sorry just saw below. Maybe not for this PR, perhaps we can integrate the theme options to pass into the `theme` prop in that table, I know that we have another story for aligning the README structure too...

Reply:
oh yeah - I'm actually working on that ticket 😅I'll put it in that PR instead once this is merged in

Reply:
ok, will resolve and look for that in the other PR!